modules/users/client/views/authentication/signin.client.view.html

```diff
@@ -6,7 +6,7 @@ <h3 class="col-md-12 text-center">Or with your account</h3>
         <div class="form-group">
           <label for="username">Username</label>
           <input type="text" id="username" name="username" class="form-control"
-               data-ng-model="credentials.username" placeholder="Username">
+               data-ng-model="credentials.username" data-ng-change="credentials.username=credentials.username.toLowerCase();" placeholder="Username">
```

Comment:
We seem to have decided not to use the `data-` prefix for angular attributes so please change this through-out the PR to just `ng-change` and so on..